README.md

```diff
@@ -284,6 +205,7 @@ def generate_embeddings_http_request(req: func.HttpRequest, embeddings: str) ->
     # TODO: Store the embeddings into a database or other storage.
     return func.HttpResponse(status_code=200)
 ```
+
```

Comment:
Should we drop the `typescript` sample code below here? I noticed that we dropped TypeScript from text completions already, and we seem to be instead focusing on C# and Python (which I think is a good choice). Also, the TypeScript sample is actually included in the `#### Python example` section, which doesn't make sense.

Reply:
I missed it probably because it was hidden in python heading, removed it now.